tree/ntuple/v7/src/RPageStorageFile.cxx

```diff
@@ -134,6 +134,28 @@ ROOT::Experimental::Detail::RPageSinkFile::CommitPageImpl(ColumnHandle_t columnH
 }
 
 
+ROOT::Experimental::RClusterDescriptor::RLocator
+ROOT::Experimental::Detail::RPageSinkFile::CommitSealedPageImpl(
+   DescriptorId_t columnId, const RPageStorage::RSealedPage &sealedPage)
+{
+   const auto bitsOnStorage = Detail::RColumnElementBase::Generate(
+      fDescriptorBuilder.GetDescriptor()
+      .GetColumnDescriptor(columnId)
+      .GetModel()
+      .GetType()).GetBitsOnStorage();
+
+   const auto bytesPacked = (bitsOnStorage * sealedPage.fNElements + 7) / 8;
```

Comment:
Doesn't the sealed page already know this information (in some in the bytes header) and/or didn't we already calculated it when reading/loading the page?  [Recalculating it wouldn't even be note worthy it was not for th fact that it seems to require an allocation to get the base information).

Reply:
reading more, isn't ```bitsOnStorage == sealedPage.fSize```?  humm .... I seems to be confused the variable name "bitsOnStorage" seems to be used to mean sometimes `number of bits of an individual elements` or `total size of the page on disk` ... humm ... I see `bits` vs `bytes` ... the factor fNElements between the 2 is counter-intuitive.